### PR TITLE
Add notebook for data fusion showcase

### PR DESCRIPTION
**Pull request by zansinergise**

### CODE REVIEW

**Comment by zcernigoj** [comment]
This is for a separate MR, just mentioning here for convenience

`notebooks/.ipynb_checkpoints/5_SH_backend_data_fusion_mean_NDVI-checkpoint.ipynb` and other "checkpoints" files in `notebooks/.ipynb_checkpoints` folder are not needed and can be removed.

Also, to avoid committing new "checkpoints" files, these 2 lines should be added to `.gitignore`:

```
.ipynb_checkpoints
*/.ipynb_checkpoints/*
```

**Comment by zcernigoj** [comment]
Please add this notebook to the list in readme

File: notebooks/5_SH_backend_data_fusion_mean_NDVI.ipynb [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Connect to openEO Platform  back-end (https://openeo.cloud) which has SH production backend implemented\n",
+    "# https://openeo.sentinel-hub.com/production/\n",
+    "# connection = openeo.connect(\"https://openeo.sentinel-hub.com/production/\")\n",
+    "connection = openeo.connect(\"http://127.0.0.1:5000\")\n",
```
Comment (zcernigoj) [inline notebooks/5_SH_backend_data_fusion_mean_NDVI.ipynb]: The URL will need to change when the data fusion will be deployed

**Review by zcernigoj — CHANGES_REQUESTED** [review]
Only the URL and adding the notebook to the list in the readme. Other than that it looks ok and works when tested with locally deployed backend :+1:

**Comment by zansinergise** [comment]
Updated the readme with a short description and changed the localhost url back to the production one, so the notebook will work normally when we merge the data fusion

**Review by zcernigoj — APPROVED** [review]

**Pull request merged**